cea/optimization/ntwOpt/NtwMain.py

```diff
@@ -9,7 +9,7 @@
 
 class ntwFeatures(object):
```

Comment:
This change did not come from me... Maybe it showed up because this is actually some updates in the master branch? The current pull request is for the contribution branch.
But yes, I started working on the network part recently. So I will keep this in mind when modifying the codes in my network branch : )